main.py

```diff
@@ -0,0 +1,75 @@
+from flask import Flask
+import random
+import csv
+import pandas as pd
+
+
+app = Flask(__name__)
+
+
+@app.route("/")
+def hello_world():
+    home_page = f"<p> Генератор паролів - /generate_password <p>" \
+                f"<p> Середнє значення csv - /calculate_average_csv </p>" \
+                f"<p> Середнє значення pandas - /calculate_average_pandas </p>"
+    return home_page
+
+
+@app.route('/generate_password')
+def generate_password():
+    pass_len = random.randint(10, 20)
+    ascii_lowercase = (97, 122)
+    ascii_uppercase = (65, 90)
+    int_symbol = (48, 57)
+    special_symbols = (33, 47)
+
+    some_lst = [ascii_lowercase, ascii_uppercase, int_symbol, special_symbols]
+    base_pass = []
+
+    counter = 0
+    while counter < pass_len:
+        if counter < len(some_lst):
+            base_pass.append(chr(random.randint(*some_lst[counter])))
+            counter += 1
+        else:
+            random_symbol = random.choice(some_lst)
+            base_pass.append(chr(random.randint(*random_symbol)))
```

Comment:
why do we need this part? we have password from 10 to 20 chars.
if counter < len(some_lst):